nuget/spec/dependabot/nuget/file_parser_spec.rb

```diff
@@ -54,13 +58,7 @@ def dependencies_from_info(deps_info)
     subject(:top_level_dependencies) { dependencies.select(&:top_level?) }
 
     context "with a .proj file" do
-      let(:files) { [proj_file] }
-      let(:proj_file) do
-        Dependabot::DependencyFile.new(
-          name: "proj.proj",
-          content: fixture("csproj", "basic2.csproj")
-        )
-      end
+      let(:project_name) { "file_parser_proj" }
```

Comment:
Moved these tests from picking and choosing file to test against to having project folders containing the expected mix of files.